portal-impl/src/resource-actions/default.xml

```diff
@@ -8,5 +8,4 @@
 	<resource file="resource-actions/documentlibrary.xml" />
 	<resource file="resource-actions/expando.xml" />
 	<resource file="resource-actions/sites.xml" />
-	<resource file="resource-actions/workflow.xml" />
```

Comment:
@dantewang 
This file can not be remove directly, it should be moved to modules where the portlets are. Please Check UpgradePortletId for the mapping of integer to portlet name.

Once you moved it out, you need to create an upgrade process to modify the values in ResourceAction table.

If this will become big, please create a new ticket to deal with it separately, then deal with others.

Thanks.
Tina.

Reply:
Hi @dantewang,

After another thought, I think we should do this: 
Remove this file and create an upgrade process to remove all related entries in ResourceAction table to make it consistent.

If we move it out to module, while upgrading, there will be conflicts with current data, which is harder comparing removal. 

Tina.